src/server/server-quiz/services/quiz-manager.js

```diff
@@ -1,10 +1,63 @@
+const urlArray = require("../clients/api-constants");
 const { Question } = require("../../db/models");
+const triviaClient = require("../clients/trivia-client");
 
 async function getAllQuestions() {
-  const questions = await Question.findAll();
+  const personalQuestions = await Question.findAll();
+
+  const triviaQuestions = await triviaClient.fetchMultipleTopics(urlArray);
+
+  const quizyTriviaQuestions = changeQuestionsStructure(triviaQuestions);
+
+  const questions = [...personalQuestions, ...quizyTriviaQuestions];
+  shuffleOptions(questions);
   return questions;
 }
 
+function changeQuestionsStructure(triviaQuestions) {
+  const newQuestionsArray = [];
+  for (let question of triviaQuestions) {
+    const newQuestion = {
+      id: null,
+      question: question.question,
+      type: "trivia",
+      topic: `${question.category}`,
+      amountOfOptions: `${question.type}`,
+    };
+
+    const options = question.incorrect_answers;
+    options.push(question.correct_answer);
+    shuffleOptions(options);
+
+    if (question.type === "multiple") {
+      newQuestion.option1 = options[0];
+      newQuestion.option2 = options[1];
+      newQuestion.option3 = options[2];
+      newQuestion.option4 = options[3];
+    }
+
+    if (question.type === "boolean") {
+      newQuestion.option1 = options[0];
+      newQuestion.option2 = options[1];
+      delete newQuestion.option3;
```

Comment:
maybe we can set option3 and option4 null or undefined instead of deleting? So we will not need to check the type with if because newQuestion.option3 = options[2] and newQuestion.option4 = options[3] will be undefined anyway.

now lines 35-36 and 40-41 are duplicating each other, but we can avoid it

Reply:
Added 
` if (options.length > 2) {
      newQuestion.option3 = options[2];
      newQuestion.option4 = options[3];
    }`
and removed duplications